.github/workflows/actions/build-core-stencil-eval/action.yml

```diff
@@ -0,0 +1,50 @@
+name: 'Build Ionic Core with Stencil Eval'
+description: 'Build Ionic Core with an Evaluation Copy of Stencil'
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16.x
+
+    - name: Install Dependencies
+      run: npm ci
+      working-directory: ./core
+      shell: bash
+    - name: Retrieve Stencil Repository
+      run: git clone https://github.com/ionic-team/stencil.git stencil-repo-tmp
+      shell: bash
+    - name: Report Stencil SHA
+      run: git rev-parse HEAD
+      working-directory: ./stencil-repo-tmp
+      shell: bash
+    - name: Build Stencil
+      run: npm ci && npm run build && npm pack
+      working-directory: ./stencil-repo-tmp
+      shell: bash
+    - name: Move the Stencil Build Artifact
+      # there isn't a great way to get the output of `npm pack`, just grab the first and hope for the best
+      run: mv $(ls ./stencil-repo-tmp/*.tgz | head -1) ./core/stencil-eval.tgz
+      shell: bash
+    - name: Install Stencil Eval
+      working-directory: ./core
+      run: npm i ./stencil-eval.tgz
```

Comment:
Are you able to do `npm install ./stencil-repo-tmp/stencil-eval.tgz` instead to avoid the extra `mv` step?

Reply:
We cannot, as the result of `npm pack` is something like `stencil-core-2.19.3.tgz`, where the version is dictated by `package.json#version`. That field will be changing over time, so we don't know it's value (the `mv` is also acting as a rename).

There _are_ ways of finding the version from `package.json` with `grep | head | awk`, but IMO that overcomplicated the pipeline a little bit (although not by much). Since we intend on getting rid of building Stencil from source (including this `mv`), I decided to keep it as is.

Reply:
Actually...I'd prefer to keep it as is. That way we can more safely rimraf the entire Stencil directory we created